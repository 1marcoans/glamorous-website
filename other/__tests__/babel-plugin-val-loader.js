import pluginTester from 'babel-plugin-tester'
import plugin from '../babel-plugin-val-loader'

pluginTester({
  plugin,
  snapshot: true,
  tests: [
    {
      fixture: require.resolve('./fixtures/val-component'),
    },
  ],
})
