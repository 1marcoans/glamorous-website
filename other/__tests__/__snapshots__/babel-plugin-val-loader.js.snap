// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. val-loader 1`] = `
"
/* eslint no-unused-vars:0 */
import val1 from './load-data.val'

const val2 = require('./load-data.val')

const array = [require('./load-data.val')]

      ↓ ↓ ↓ ↓ ↓ ↓

/* eslint no-unused-vars:0 */
const val1 = 'Hello world!';


const val2 = 'Hello world!';

const array = ['Hello world!'];
"
`;
